masq/src/commands/financials_command.rs

```diff
@@ -797,84 +692,89 @@ impl FinancialsCommand {
     }
 
-    fn masq_values_range_regex() -> Regex {
-        Regex::new("(^(-?\\d+\\.?\\d*)\\s*-\\s*(-?\\d+\\.?\\d*))|(^-?\\d+\\.?\\d*)$")
-            .expect("wrong regex")
-    }
-
     fn extract_individual_masq_values(
-        masq_range_str: &str,
-        masq_values_range_regex: Regex,
+        masq_in_range_str: &str,
+        masq_values_in_range_regex: Regex,
     ) -> Result<(String, Option<String>), String> {
         fn handle_captures(captures: Captures) -> (Option<String>, Option<String>) {
             let fetch_group = |idx: usize| FinancialsCommand::single_capture(&captures, idx);
             match (fetch_group(2), fetch_group(3)) {
                 (Some(second), Some(third)) => (Some(second), Some(third)),
-                (None, None) => (
-                    Some(fetch_group(4).expect("the regex is wrong if it allows this panic")),
-                    None,
-                ),
-                (first, second) => unreachable!(
-                    "the regex was designed not to allow this: {:?}, {:?}",
-                    first, second
-                ),
+                (None, None) => {
+                    let four = fetch_group(4).expect("the regex is wrong if it allows this panic");
+                    (Some(four), None)
+                }
+                (x, y) => {
+                    unreachable!(
+                        "the regex was designed not to allow '{:?}' for the second and \
+                     '{:?}' for the third capture group",
+                        x, y
+                    )
+                }
             }
         }
 
-        match masq_values_range_regex
-            .captures(masq_range_str)
+        match masq_values_in_range_regex
+            .captures(masq_in_range_str)
             .map(handle_captures)
         {
             Some((Some(first), Some(second))) => Ok((first, Some(second))),
             Some((Some(first), None)) => Ok((first, None)),
-            _ => Err("Second range in improper format".to_string()),
+            _ => Err(format!(
+                "Second range '{}' in improper format",
+                masq_in_range_str
+            )),
         }
     }
 
-    pub fn parse_masq_range_to_gwei<N>(range_str: &str) -> Result<(N, N, String, String), String>
+    pub fn parse_masq_range_to_gwei<N>(range: &str) -> Result<(N, N, String, String), String>
     where
         N: FromStr<Err = ParseIntError> + TryFrom<i64> + Mul<Output = N> + CheckedMul + Display,
         <N as TryFrom<i64>>::Error: Debug,
     {
-        let (first, second_opt) =
-            Self::extract_individual_masq_values(range_str, Self::masq_values_range_regex())?;
-        let first_as_num = Self::process_optionally_fragmentary_number(&first)?;
-        let second_as_num = if let Some(second) = second_opt.as_ref() {
-            Self::process_optionally_fragmentary_number(second)?
+        let regex = Regex::new("^((-?\\d+\\.?\\d*)\\s*-\\s*(-?\\d+\\.?\\d*))|(-?\\d+\\.?\\d*)$")
```

Comment:
Isn't there a facility in Rust where you can specify a string like `r#"`...blah...`"` and not have to put in double backslashes?